docs/framework/wpf/advanced/use-automatic-layout-overview.md

```diff
@@ -42,13 +42,13 @@ Auto Resizable Button
 ## Automatic Layout and Coding Standards  
  Using the automatic layout approach requires a set of coding and design standards and rules to produce a fully localizable  [!INCLUDE[TLA2#tla_ui](../../../../includes/tla2sharptla-ui-md.md)]. The following guidelines will aid your automatic layout coding.  
   
-|Coding Standards|Description|  
-|----------------------|-----------------|  
-|Do not use absolute positions.|-   Do not use <xref:System.Windows.Controls.Canvas> because it positions elements absolutely.<br />-   Use <xref:System.Windows.Controls.DockPanel>, <xref:System.Windows.Controls.StackPanel>, and <xref:System.Windows.Controls.Grid> to position controls.<br />-   For a discussion about various types of panels, see [Panels Overview](../../../../docs/framework/wpf/controls/panels-overview.md).|  
-|Do not set a fixed size for a window.|-   Use <xref:System.Windows.Window.SizeToContent%2A>.<br />-   For example:<br /><br /> [!code-xaml[LocalizationGrid#2](../../../../samples/snippets/csharp/VS_Snippets_Wpf/LocalizationGrid/CS/Pane1.xaml#2)]|  
-|Add a <xref:System.Windows.FrameworkElement.FlowDirection%2A>.|<ul><li>Add a <xref:System.Windows.FrameworkElement.FlowDirection%2A> to the root element of your application.</li><li>[!INCLUDE[TLA2#tla_winclient](../../../../includes/tla2sharptla-winclient-md.md)] provides a convenient way to support horizontal, bidirectional, and vertical layouts. In presentation framework,   the <xref:System.Windows.FrameworkElement.FlowDirection%2A> property can be used to define layout. The flow-direction patterns are:<br /><br /> <ul><li><xref:System.Windows.FlowDirection.LeftToRight> (LrTb) — horizontal layout for Latin, East Asian, and so forth.</li><li><xref:System.Windows.FlowDirection.RightToLeft> (RlTb) — bidirectional for Arabic, Hebrew, and so forth.</li></ul></li></ul>|  
-|Use composite fonts instead of physical fonts.|<ul><li>With composite fonts, the <xref:System.Windows.Controls.Control.FontFamily%2A> property does not need to be localized.</li><li>Developers can use one of the following fonts or create their own.<br /><br /> <ul><li>Global User Interface</li><li>Global San Serif</li><li>Global Serif</li></ul></li></ul>|  
-|Add xml:lang.|-   Add the `xml:lang` attribute in the root element of your [!INCLUDE[TLA2#tla_ui](../../../../includes/tla2sharptla-ui-md.md)], such as `xml:lang="en-US"` for an English application.<br />-   Because composite fonts use `xml:lang` to determine what font to use, set this property to support multilingual scenarios.|  
+| Coding Standards | Description |
```

Comment:
from what I understood from Curt, usually the problem with tables is that we have html elements on them. not sure what to do about that.

Reply:
Here's the Markdig-published page of the original document: https://review.docs.microsoft.com/en-us/dotnet/framework/wpf/advanced/use-automatic-layout-overview?branch=markdig-migration. The problem here, from what I've seen in the issues I've addressed, is that code inside tables doesn't play at all well. The best option here is to not use a table format, but use a list/nested list outside of a table.

Reply:
@mairaw @rpetrusha The markdig system also has a new table system (called [grid table](https://github.com/lunet-io/markdig/blob/master/src/Markdig.Tests/Specs/GridTableSpecs.md)) that lets you put in markdown. So this issue of having to use html lists in the table goes away. This may also help the include problem.